src/flashbang/core/FlashbangApp.ts

```diff
@@ -174,6 +176,10 @@ export default class FlashbangApp {
     }
 
     protected onMouseWheelEvent(e: WheelEvent): void {
+        // kkk dispatch WheelEvent to NGL
+        if (Mol3DGate.scope && Mol3DGate.scope.isOver3DCanvas) {
```

Comment:
This should be done in the onMouseWheelEvent function of GameMode

Reply:
I removed this here and wrote it again in GameMode.ts